ihatemoney/templates/invitation_mail.j2

```diff
@@ -1,3 +1,5 @@
+
```

Comment:
I don't believe we need the extra spaces here. Any reason you added them?

Reply:
fixed